master/reference/calicoctl/commands/node/run.md

```diff
@@ -104,6 +130,83 @@ CALICO_LIBNETWORK_ENABLED is true - start libnetwork service
 Calico node started successfully
 ```
 
+#### IP Autodetection method examples
+
+The node resource includes IPv4 and IPv6 routing IP addresses that should
+match those on one of the host interfaces.  These IP addresses may be 
+configured in advance by configuring the node resource prior to starting the
+calico/node service, or alternatively the addresses may either be explicitly 
+specified, or autodetected through options on the `calicoctl run` command.
+
+There are different autodetection methods available and you should use the one
+best suited to your deployment.  If you are able to explicitly specify the IP 
+addresses, that is always preferred over autodetection. This section describes
+the available methods for autodetecting the hosts IP addresses.
+
+An IPv4 address is always required, and so if no address was previously
+configured in the node resource, and no address was specified on the CLI, then
+we will attempt to autodetect an IPv4 address.  An IPv6 address, however, will 
+only be autodetected when explicitly requested.
+
+To force autodetection of an IPv4 address use the option `--ip=autodetect`.  To
+force autodetection of an IPv6 address use the option `--ip6=autodetect`. 
+
+To set the autodetection method for IPv4, use the `--ip-autodetection-method` option.
+To set the autodetection method for IPv6, use the `--ip6-autodetection-method` option.
+
+>  Note, if you are starting the calico/node container directly (and not using the
```

Comment:
we have been printing note as: 
```
> **NOTE**
> blah blah ,meep
> more blah
```
So it looks consistent in the html rendering.